pages/[wallet]/index.tsx

```diff
@@ -728,8 +731,24 @@ export const ContractTable: React.FC<ContractTableProps> = ({
 
   const wallet = useSingleQueryParam("wallet") || "dashboard";
 
-  if (!combinedList.length) {
-    return <NoContracts />;
+  if (combinedList.length === 0) {
```

Comment:
what happens during loading? it will be length 0, no? so when loading we just throw u over to `/contracts`?